core/src/main/scala/org/broadinstitute/dsde/rawls/submissions/SubmissionsService.scala

```diff
@@ -124,9 +128,12 @@ object SubmissionsService {
       genomicsServiceConstructor,
       config,
       workspaceRepository,
-      workspaceSettingRepository
+      workspaceSettingRepository,
+      entityServiceConstructor
     )
 
+  final val terraCreatedSetToDeleteAttribute = AttributeName("default", "deleteTerraCreatedSet")
```

Comment:
I'm not the biggest fan of this naming, which seems to mix what it is (workflow-based set) with something we may or may not do with it (delete it).

```suggestion
  final val workflowSubmissionSetAttribute = AttributeName("default", "workflowSubmissionSet")
```

Reply:
I've updated to a slightly different tack that doesn't require dumb magic words